source/NurMicroApi.c

```diff
@@ -12,6 +12,7 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTH
 WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */
 
+#include <string.h>
```

Comment:
I think this should be included only if HAVE_NUR_MEMCPY is not defined in NurApiConfig.h
And all memcpy's should be replaced with nurMemcpy